manifests/cf-manifest/scripts/rotate-vars-store-secrets.rb

```diff
@@ -24,6 +24,9 @@ def parse_args
   parser.on("--preserve VAR", "variables to not rotate") { |v|
     options[:vars_to_preserve] << v
   }
+  parser.on("--rotate VAR", "variables to do rotate. Will rotate all if not set.") { |v|
+    options[:vars_to_rotate] = (options[:vars_to_rotate] || []) << v
```

Comment:
would be nicer to initialise vars_to_rotate on line 13, as we do it with vars_to_preserve

Reply:
we can leave this as is